exercises/3_5/Parte4.html

```diff
@@ -0,0 +1,19 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Creepster|Text+Me+One" rel="stylesheet">
+    <title>A Creepy Story</title>
+    <link href="parte4.css" rel="stylesheet">
+  </head>
+  <body>
+    <h1>The Basement</h1>
+    <p id="story">I knew I shouldn't go there, in that dank, cold little place. But I wanted to find my friend, and he was hiding in the basement.</p>
+    
+    <button>Get In The Mood</button>
+    <div class="textEntry">Type your name here, if you dare: <input type="text" id="name" autocomplete="off"></div>
+    <button>Please, Please Don't Click Me</button>
+    <button>Tell Me What Happened!</button>
+    <audio id="audio" src="http://reboot-me.com/ouch.mp3" type="audio/mpeg"></audio>
+  </body>
+  <script src="Parte4.js"></script>
+</html>
```

Comment:
O que você acha de configurar o seu editor de código para adicionar automaticamente uma linha em branco ao salvar o arquivo?

Reply:
Por exemplo: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files